### PR TITLE
Update for move to dist based repo naming

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: docker/clients/fedora20/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM fedora:20
+FROM fedora:22
```
Comment (jlsherrill) [inline docker/clients/fedora20/Dockerfile]: unrelated changes?

Comment (jlsherrill) [inline docker/clients/fedora20/Dockerfile]: or just updating to supported fedora client?

**Comment by jlsherrill** [comment]
Haven't tested, but looks good to me if the dockerfile change is intended.

File: docker/clients/fedora20/Dockerfile (hunk 1 above)
Comment (ehelms) [inline docker/clients/fedora20/Dockerfile]: I had to update the repo location in the Dockerfile and realized they were still set to f20 and f21 which won't work for nightly so I updated them to reflect the repos that will actually exist.

**Comment by jlsherrill** [comment]
ACK
